### DISCUSSION TITLE
BarcodeScanner.setZoomRatio() missing from version 5.3.0 (latest)

### DISCUSSION THREAD

**Discussion by amarinp** [capawesome-team/capacitor-mlkit#114] [Q&A] [answered]
Thank you for this plugin.
On your example you use @capacitor-mlkit/barcode-scanning version 5.3.0-dev.966ebeb.1703101506 on capacitor 5.4.1
I use  version 5.3.0 (latest) on capacitor 5.4.0. I am missing BarcodeScanner.setZoomRatio(), BarcodeScanner.getMinZoomRatio(), BarcodeScanner.getMaxZoomRatio(). I notice they are missing from java and Swift source. Please tell me how do I get them? or are they removed?

**Comment by robingenz** [comment, marked as answer]
The zoom feature will be available in version 5.4.0, see https://github.com/capawesome-team/capacitor-mlkit/pull/113. For now, you can only use the dev version.

**Reply by amarinp** [reply]
Thank you for your answer and again for this plugin. Happy Holidays.
